app/Auth/MissingAttributeException.php

```
@@ -2,11 +2,19 @@
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
 class MissingAttributeException extends \Exception
 {
     public function __construct(string $attribute)
     {
         $message = "Missing attribute: $attribute";
         parent::__construct($message);
     }
+
+    public function render(Request $request): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)